### ISSUE TITLE
Incorrect keystroke triggers camera

### ISSUE THREAD

**Issue by pmartz** [chigkim/VOCR#34]
I have an AppleScript that I frequently use with Google Chrome and trigger it with Control+Shift+3. Sometimes, but not always, this triggers a VOCR Camera Capture as if I had pressed Command+Control+Shift+C. VOCR gives me the countdown and snaps a photo, then VOCR asks how I want to process or describe the image. This is incorrect. VOCR should only execute Camera Capture when I press Command+Control+Shift+C. If I exit VOCR, my Control+Shift+3 shortcut again correctly invokes my Google Chrome AppleScript. I have checked VOCR/Settings/Shortcuts and verified that Capture Camera is configured for ⌃⇧⌘C. I hope you can investigate and resolve this issue. I am not a programmer. Thanks.
[VOCR.txt](https://github.com/user-attachments/files/18117466/VOCR.txt)

**Comment by chigkim** [comment]
Can you check your VOCR shortcuts in the settings? You might have changed it by mistake. When I press control+shift+3 with VOCR on, MacOS makes a little sounds that indicates I pressed a shortcut that does nothing.

**Comment by pmartz** [comment]
Okay. I figured out what’s going wrong. I trigger an AppleScript with Control+Shift+3. That AppleScript emits a keystroke: Control+Command+C, to be interpreted in Google Docs. But if I mistakenly continue to hold the Shift key, Shift is added to the emitted command, and that invokes the VOCR Camera Capture.

Sorry I bothered you with this, but it helped for me to think through the problem. I can work around by modifying the VOCR Camera Capture shortcut.
